src/gui/ocssharejob.cpp

```diff
@@ -118,6 +118,17 @@ void OcsShareJob::setPermissions(const QString &shareId,
     start();
 }
 
+void OcsShareJob::setLabel(const QString &shareId, const QString &label)
```

Comment:
I would probably use a name more like `setLabelByShareId`
what do you think ?

Reply:
@mgallien I guess this will be confusing and does not represent the real situation. The label is being set for share with `shareId`, not to something else. So, it's more of a `setLabelForShareId`. Having said that, I would keep the name as simple `setLable`. Other functions in the interface also follow this convention. So, we either follow it too for new methods, or, we rename every other function that does similar job.

Reply:
OK I had missed that convention `setLabel` is fine for me